common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -739,6 +739,28 @@ bool mon_wlan_hal_dwpal::sta_beacon_11k_request(const SBeaconRequest11k &req, in
         cmd += " ssid=" + req_ssid;
     }
 
+    // use ap_ch_report in the request.
+    if (req.use_optional_ap_ch_report) {
+        std::stringstream ch_report_ss;
+        uint8_t current_channel = 0;
+        for (; current_channel < req.use_optional_ap_ch_report - 1; ++current_channel) {
+            ch_report_ss << +req.ap_ch_report[current_channel] << ",";
+        }
+        ch_report_ss << +req.ap_ch_report[current_channel];
+
+        cmd += " ap_ch_report=" + ch_report_ss.str();
+    } else {
+        LOG(WARNING) << "optinal channel report was not set (might be also ok)";
+    }
+
+    // report detail: rep_detail=
+    // this value should be set by default to 2 if not stated otherwise.
+    // however hostap does not set it to 2. So we add it hardcoded here
+    // note: currently there is no field in SBeaconRequest11k that holds this optional field
+    cmd += " rep_detail=2";
+
+    LOG(DEBUG) << __func__ << " the command: " << cmd;
```

Comment:
Remove

Reply:
Remove what?
Should I remove the only debug log message along the way that shows me what was actually sent? If it was me to decide I would make it an `INFO` message. This line tells a story in the log. I think that when you read the log, you should understand what was happening not based only on errors (the way it was up till this line), you should have beacons that signal to you "I am here, everything is fine". 
Of course that a balance is needed: if the line I added creates 10th or 100th of these signals along the log, this message shouldn't be. I think that this specific line is correct as it prints only beacon query messages.